stable/opa/templates/NOTES.txt

```diff
@@ -53,3 +53,6 @@ spec:
 EOF
 
 kubectl -n opa-example create -f ingress-bad.yaml
+
+If you want to turn off authz for debugging purposes, you can do so by upgrading the chart like so:
+helm upgrade {{ .Release.Name }} stable/opa --reuse-values --set authz.enabled=false
```

Comment:
Let's add the bit from above about querying the OPA here (taking the lines that were deleted from above and moving them down here.)

```
You can query OPA to see the policies it has loaded (you will need to turn off authz as described above):

export OPA_POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "opa.fullname" . }}" -o jsonpath="{.items[0].metadata.name}")

kubectl port-forward $OPA_POD_NAME 8080:443

curl -k -s https://localhost:8080/v1/policies | jq -r '.result[].raw'
```